specs/vscode_extensions.table

```diff
@@ -0,0 +1,17 @@
+table_name("vscode_extensions")
+description("Lists all vscode extensions.")
+schema([
+    Column("id", TEXT, "Extension id"),
```

Comment:
Interesting that this is the `id`. Elsewhere in vscode, this data is the `name`. But :shrug:

I think it'd be good to include the `uuid` as well.

Reply:
Depends which json. It looks like it's `id` in `.extensions.json` but `name` in `foo/package.json`. Similarly, `uuid` vs `__metadata.id`.

Given the internal inconsistency, I think we should just pick id/uuid or name/id. As long as they're both exposed I think we'll be okay